src/modules/wallet/helpers/transaction-helper.js

```diff
@@ -9,7 +11,7 @@ export const getProgramName = transaction => {
   if (transaction.investmentProgram) {
     return (
       <div className="wp-transaction__program">
-        {transaction.investmentProgram.title}
+        <Link to={`${PROGRAMS_ROUTE}/${transaction.investmentProgram.id}`}>{transaction.investmentProgram.title}</Link>
```

Comment:
1) Use function "replaceParams" from utils if you want to use routing.
2) Do not underline links on hover